api.bs

```diff
@@ -1240,6 +1240,9 @@ and given <a dictionary lt=PrivateAttributionImpressionOptions>|options|</a>:
         throw a {{RangeError}}.
     1.  Clamp |options|.{{PrivateAttributionImpressionOptions/lifetimeDays}} to
         the [=user agent=]'s upper limit.
+    1.  If the [=list/size=] of
+        |options|.{{PrivateAttributionImpressionOptions/conversionSites}} is
+        greater than an [=implementation-defined=] maximum value, throw a {{RangeError}}.
```

Comment:
```suggestion
    1.  If the [=list/size=] of
        |options|.{{PrivateAttributionImpressionOptions/conversionSites}} is
        greater than an [=implementation-defined=] maximum value, throw a {{RangeError}}.
        This maximum MUST be at least 10.
```

The discussion at the meeting established that a minimum was necessary for interoperability and that 10 was more than enough (apparently ARA set this at 3, which caused some problems).